### PR TITLE
CI: enable Travis static checks

### PR DESCRIPTION
**Pull request by grahamwhaley**
Set up the config file and .ci subdir files to enable Travis to perform static
checks on PRs to this repo.

### CODE REVIEW

**Comment by grahamwhaley** [comment]
ptal - I suspect we could:
 - optimise the Travis file
 - add more checks, like build checks for `cmd/*`

We can always land as is and improve upon...

File: .ci/static-checks.sh [hunk 1]
```diff
@@ -1,14 +1,13 @@
...
 # SPDX-License-Identifier: Apache-2.0
-#
 
-set -o errexit
-set -o nounset
-set -o pipefail
+set -e
+
+cidir=$(dirname "$0")
+source "${cidir}/lib.sh"
```
Comment (chavafg) [inline .ci/static-checks.sh:10]: seems like we need to add the `lib.sh` in the `.ci` directory

File: .travis.yml [hunk 2]
```diff
@@ -0,0 +1,24 @@
+#
+# Copyright 2017 HyperHQ Inc.
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+sudo: required
+dist: xenial
```
Comment (devimc) [inline .travis.yml:8]: why not bionic ?

File: .ci/static-checks.sh (hunk 1 above)
Comment (grahamwhaley) [reply .ci/static-checks.sh:10]: doh, thanks. I forgot to `git add` it .... I did add and test locally....

File: .travis.yml (hunk 2 above)
Comment (grahamwhaley) [reply .travis.yml:8]: I copied from another repo - I'll go with whatever - so, let me switch it to bionic... np

Comment (grahamwhaley) [reply .travis.yml:8]: for ref then, I think there are still a few repos using xenial in their Travis. iirc, I copied from the agent (I wanted a 'simple' template).
https://github.com/search?q=org%3Akata-containers+xenial&type=Code

Comment (chavafg) [reply .travis.yml]: yeah, I think all other repos still use xenial, would be good to later make the update in those places.

**Comment by grahamwhaley** [comment]
gentle nudge here - the PR is pretty simple, so easy to pick up and replicate - but, let me know if there is anything I can fix up to push this on. @jcvenegas @chavafg

**Comment by fidencio** [comment]
@grahamwhaley, obs-packaging-ci is still failing though.

**Comment by grahamwhaley** [comment]
@fidencio - ack. I suspect it is unrelated to this PR (I cannot think how it would be related), but thought I'd let @jcvenegas look into it :-)

**Comment by chavafg** [comment]
tried to look at logs, but build not found, so rerunning again.

**Pull request merged**